src/PostgREST/CLI.hs

```diff
@@ -0,0 +1,351 @@
+{-|
+Module      : PostgREST.CLI
+Description : Command line interface.
```

Comment:
If we still keep the distinction between executable and library - this module should really be part of the executable, no? I don't think we test any of that in the spec tests, only in the io tests - that seems like a sign that all of this can only be used in the executable.